events/community-meeting.md

```diff
@@ -0,0 +1,73 @@
+# Kubernetes Weekly Community Meeting
+
+We have PUBLIC and RECORDED [weekly meeting](https://zoom.us/my/kubernetescommunity) every Thursday at 6pm UTC (1pm EST / 10am PST)
+ 
+Map that to your local time with this [timezone table](https://www.google.com/search?q=1800+in+utc)
+
+See it on the web at [calendar.google.com](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles) , or paste this [iCal url](https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics) into any iCal client.
+
+To be added to the calendar items, join the Google group
+[kubernetes-community-video-chat](https://groups.google.com/forum/#!forum/kubernetes-community-video-chat) for further instructions.
+
+All meetings are archived on the [Youtube Channel](https://www.youtube.com/watch?v=onlFHICYB4Q&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ)
+
+Quick links:
+
+- [Agenda document](https://k8s.devstats.cncf.io/)
+- [Zoom meeting link](https://zoom.us/my/kubernetescommunity)
+
+## Purpose
+
+The purpose of the community meeting is the dissemination of information throughout the project. 
+Large changes in an area of Kubernetes that affect groups of people should be mentioned here. 
+
```

Comment:
Alternate wording:  The Kubernetes community meeting is intended to provide a holistic overview of community activities, critical release information, and governance updates. It also provides a forum for discussion on project-level concerns that might need a wider audience than a single special interest group (SIG).